Jagrati/settings/base.py

```diff
@@ -118,3 +119,12 @@
 
 
 AUTH_USER_MODEL = 'accounts.User'
+
+AUTHENTICATION_BACKENDS=['django.contrib.auth.backends.ModelBackend',
```

Comment:
Put the `AUTHENTICATION_BACKENDS` after `WSGI_APPLICATION` and with proper indentation like other variables.